OpenXmlPowerTools/HtmlToWmlCssParser.cs

```diff
@@ -1371,67 +1144,67 @@ private int GetHueValue(CssTerm t)
-            if (m_type == CssTermType.Hex)
+            if (Type == CssTermType.Hex)
             {
-                if ((m_val.Length == 7 || m_val.Length == 4) && m_val.StartsWith("#"))
+                if ((Value.Length == 7 || Value.Length == 4) && Value.StartsWith("#"))
                 {
-                    hex = m_val.Substring(1);
+                    hex = Value.Substring(1);
                 }
-                else if (m_val.Length == 6 || m_val.Length == 3)
+                else if (Value.Length == 6 || Value.Length == 3)
                 {
-                    hex = m_val;
+                    hex = Value;
                 }
             }
-            else if (m_type == CssTermType.Function)
+            else if (Type == CssTermType.Function)
             {
-                if ((m_function.Name.ToLower().Equals("rgb") && m_function.Expression.Terms.Count == 3)
-                    || (m_function.Name.ToLower().Equals("rgba") && m_function.Expression.Terms.Count == 4)
+                if ((Function.Name.ToLower().Equals("rgb") && Function.Expression.Terms.Count == 3)
+                    || (Function.Name.ToLower().Equals("rgba") && Function.Expression.Terms.Count == 4)
                     )
                 {
                     int fr = 0, fg = 0, fb = 0;
-                    for (var i = 0; i < m_function.Expression.Terms.Count; i++)
+                    for (var i = 0; i < Function.Expression.Terms.Count; i++)
                     {
-                        if (m_function.Expression.Terms[i].Type != CssTermType.Number)
+                        if (Function.Expression.Terms[i].Type != CssTermType.Number)
                         {
                             return Color.Black;
                         }
                         switch (i)
                         {
                             case 0:
-                                fr = GetRGBValue(m_function.Expression.Terms[i]);
+                                fr = GetRGBValue(Function.Expression.Terms[i]);
                                 break;
 
                             case 1:
-                                fg = GetRGBValue(m_function.Expression.Terms[i]);
+                                fg = GetRGBValue(Function.Expression.Terms[i]);
                                 break;
 
                             case 2:
-                                fb = GetRGBValue(m_function.Expression.Terms[i]);
+                                fb = GetRGBValue(Function.Expression.Terms[i]);
                                 break;
                         }
                     }
                     return Color.FromArgb(fr, fg, fb);
                 }
-                else if ((m_function.Name.ToLower().Equals("hsl") && m_function.Expression.Terms.Count == 3)
-                  || (m_function.Name.Equals("hsla") && m_function.Expression.Terms.Count == 4)
+                else if ((Function.Name.ToLower().Equals("hsl") && Function.Expression.Terms.Count == 3)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/Codeuctivity/Open-Xml-PowerTools/pullRequest?prid=5308324)